### PR TITLE
removing telemetry logs for m105

### PR DESCRIPTION
**Pull request by SatishThareja**

### CODE REVIEW

File: Tasks/VsTest/VSTest.ps1 [hunk 1]
```diff
@@ -41,7 +41,6 @@ import-module "Microsoft.TeamFoundation.DistributedTask.Task.CodeCoverage"
 
 . $PSScriptRoot\Helpers.ps1
 
-Write-Host "##vso[task.logissue type=warning;TaskName=VSTest]"
```
Comment (acesiddhu) [inline Tasks/VsTest/VSTest.ps1]: there are some in comd let as well

**Comment by acesiddhu** [comment]
:shipit:

**Comment by SatishThareja** [comment]
Testing done:
1. Publish test results task
2. Vstest task - PS and TS
3. DeployTestAgent Task
4. RunFunctionalTestatsk

**Comment by allendm-msft** [comment]
#1920

**Pull request merged**